bamboo/unit-tests.sh

```diff
@@ -8,4 +8,4 @@ set -ex
 docker ps -a ## Show running containers for output logs
 
 docker exec -i ${container_id}\_build_env_1 /bin/bash -c "cd $UNIT_TEST_BUILD_DIR && npm run db:local:reset"
-docker exec -i ${container_id}\_build_env_1 /bin/bash -c "cd $UNIT_TEST_BUILD_DIR && npm run test:ci"
+docker exec -i ${container_id}\_build_env_1 /bin/bash -c "cd $UNIT_TEST_BUILD_DIR && npm run test:coverage"
```

Comment:
We should have an arch (or team if there's a hurry)  briefing re: these changes before merging.